### PR TITLE
Bug 1203762 - [Contacts] Images disappear when user enables and disab…

### PR DESCRIPTION
**Pull request by arcturus**
…les order by last button inside the contacts list. part-2 r=ferjm

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b9f632130090e9c27f9847f1fa08e9a686602278

File: apps/communications/contacts/js/views/list.js [hunk 1]
```diff
@@ -1117,9 +1117,13 @@ contacts.List = (function() {
     }
 
     if (img) {
-      delete img.dataset.group;
-      img.style.backgroundPosition = img.dataset.backgroundPosition || '';
-      setImageURL(img, photo, asClone);
+      // Update just in case we are reusing the row for a different user.
```
Comment (ferjm) [inline apps/communications/contacts/js/views/list.js]: uber-nit: "Update only if we are..."

Comment (arcturus) [inline apps/communications/contacts/js/views/list.js]: Challenge accepted

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=39c431992fab4d158fe68e3f38834fcc277cb3e7

**Pull request merged**